rdwatch/core/models/site_observation.py

```diff
@@ -96,25 +103,46 @@ def bulk_create_from_site_evaluation(
 
             constellation = constellation_map.get(feature.properties.sensor_name, None)
 
-            assert isinstance(feature.parsed_geometry, Polygon | MultiPolygon)
+            assert isinstance(feature.parsed_geometry, Polygon | MultiPolygon | Point)
 
-            geometry = (
-                feature.parsed_geometry
-                if isinstance(feature.parsed_geometry, MultiPolygon)
-                else MultiPolygon([feature.parsed_geometry])
-            )
-            for i, polygon in enumerate(geometry):
+            point = False
+            if isinstance(feature.parsed_geometry, Point):
+                point = True
+            if point is False:
+                geometry = (
+                    feature.parsed_geometry
+                    if isinstance(feature.parsed_geometry, MultiPolygon)
+                    else MultiPolygon([feature.parsed_geometry])
+                )
+                for i, polygon in enumerate(geometry):
+                    if feature.properties.current_phase:
+                        phase = feature.properties.current_phase[i]
+                    else:
+                        phase = 'Unknown'
+
+                    site_observations.append(
+                        cls(
+                            siteeval=site_eval,
+                            label=label_map[phase],
+                            score=feature.properties.score,
+                            geom=polygon,
+                            constellation=constellation,
+                            spectrum=None,
+                            timestamp=feature.properties.observation_date,
+                        )
+                    )
+            if point:
+                point = feature.parsed_geometry
```

Comment:
Couple things here -

1) `if not point` is more pythonic than `if point is False` 
2) I think this reads cleaner if the negative case comes after. To make it easier in the github editor I just swapped out the boolean to represent whether it's a polygon instead of a point, but either way is good i think
3) Isn't the `point` variable uneeded here (which makes the 1st point here moot)? I believe you can just check `isinstance` directly and get the same outcome.

```suggestion
            is_polygon = False
            if isinstance(feature.parsed_geometry, Polygon | MultiPolygon):
                geometry = (
                    feature.parsed_geometry
                    if isinstance(feature.parsed_geometry, MultiPolygon)
                    else MultiPolygon([feature.parsed_geometry])
                )
                for i, polygon in enumerate(geometry):
                    if feature.properties.current_phase:
                        phase = feature.properties.current_phase[i]
                    else:
                        phase = 'Unknown'

                    site_observations.append(
                        cls(
                            siteeval=site_eval,
                            label=label_map[phase],
                            score=feature.properties.score,
                            geom=polygon,
                            constellation=constellation,
                            spectrum=None,
                            timestamp=feature.properties.observation_date,
                        )
                    )
            else:
                point = feature.parsed_geometry
```